qiskit_neko/tests/nature/test_ground_state_solver.py

```diff
@@ -27,6 +30,10 @@
 class TestGroundStateSolvers(base.BaseTestCase):
     """Test the use of the execute() method in qiskit-terra."""
 
+    @unittest.skipIf(
+        tuple(map(int, qiskit_nature.__version__.split("."))) < (0, 7, 0),
```

Comment:
Qiskit nature doesn't do pre, post releases does it? This would error if the there version string were 0.8.0rc1 for example. [Pep 440](https://peps.python.org/pep-0440/) documents valid version strings and it's not always just 3 period separate integers.

Actually now that I've thought about it a bit more, this also would break nature's ci if they're setting a dev version with the git sha1 from editable installs like we do in qiskit. As the [dev versions have non int characters](https://peps.python.org/pep-0440/#development-release-separators). For example on qiskit right now it's something like: `0.25.2.dev0+90a20e9`

Reply:
Yes, I was cutting some corners assuming the 3 period separate integers. I have updated it to check the first 2 integers, which should always be there (this is also the way it was being done in `qiskit_algorithms` now that I look at it). This is not accounting for the possibility of having epoch versions (as defined in PEP 440), but I don't think we would be using them in Nature so it felt like a bit of an overkill.